### PR TITLE
[17.0][MIG] mail_activity_done: Migration to 17.0

### PR DESCRIPTION
**Pull request by vvrossem**
Migration done from 16.0 to 17.0.
However, 16.0 doesn't contain the following fix merged in 15.0: 
[[15.0] [FIX] mail_activity_done: Remove broken and apparently unused js file #994](https://github.com/OCA/social/pull/994)
The PR [[16.0][FIX] mail_activity_done: Remove broken and apparently unused js file #1386](https://github.com/OCA/social/pull/1386) ports it from 15.0 to 16.0.

This PR ports it from 15.0 to 17.0

### CODE REVIEW

**Comment by vvrossem** [comment]
Supersedes https://github.com/OCA/social/pull/1346

File: mail_activity_done/models/mail_activity.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class MailActivity(models.Model):
         compute="_compute_state",
         search="_search_state",
     )
-    date_done = fields.Date("Completed Date", index="btree", readonly=True)
+    date_done = fields.Date("Completed Date", index="btree")
```
Comment (yankinmax) [inline mail_activity_done/models/mail_activity.py:19]: Why are you removing `readonly=True` here and add it to the view?

Comment (vvrossem) [reply mail_activity_done/models/mail_activity.py:19]: From the [Migration to version 17.0](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0) guidelines: 

> If a field is marked as readonly=True on the model, it won't be possible to import it through the Odoo import tool, so avoid it as possible and define the `readonly` attribute on the views instead.

**Review by StefanRijnhart — APPROVED** [review]
Looking good! Code review and some functional testing.

**Review by yankinmax — APPROVED** [review]

**Comment by yankinmax** [comment]
@vvrossem thanks for the fixes porting!

**Review by hbrunn — APPROVED** [review]

**Pull request merged**